src/main/java/com/hubsante/model/report/ErrorCode.java

```diff
@@ -27,7 +27,8 @@ public enum ErrorCode {
   SCHEMA_NOT_FOUND(301, "SCHEMA_NOT_FOUND"),
   EXPIRED_MESSAGE_BEFORE_ROUTING(400, "EXPIRED_MESSAGE_BEFORE_ROUTING"),
   DEAD_LETTER_QUEUED(500, "DEAD_LETTER_QUEUED"),
-  UNROUTABLE_MESSAGE(501, "UNROUTABLE_MESSAGE");
+  UNROUTABLE_MESSAGE(501, "UNROUTABLE_MESSAGE"),
+  MESSAGE_CONVERSION_ERROR(600, "MESSAGE_CONVERSION_ERROR");
```

Comment:
Top ! Tu peux en profiter pour rajouter aussi ça 
```suggestion
  MESSAGE_CONVERSION_ERROR(600, "MESSAGE_CONVERSION_ERROR");
  INTERNAL_CLIENT_ERROR(700, "INTERNAL_CLIENT_ERROR");
```
Et on pourra dire aux éditeurs qu'ils peuvent renvoyer ce code s'ils font des messages d'erreur
D'ailleurs, ça me fait penser que dans les travaux de conversion entre versions, il faudra bien penser aux messages techniques (Ack, info) aussi et pas que les messages fonctionnels !